composer.json

```diff
@@ -35,5 +35,8 @@
     },
     "bin": [
         "parallel-lint"
-    ]
+    ],
+    "scripts": {
+        "tests": "@php vendor/bin/tester -p php tests"
+    }
```

Comment:
It is a good practice to add a `scripts-descriptions` section: https://getcomposer.org/doc/articles/scripts.md#custom-descriptions-